test/integration/blob_integration_test.exs

```diff
@@ -57,33 +57,50 @@ defmodule Azurex.BlobIntegrationTests do
                blob_name,
                @sample_file_contents,
                "text/plain",
-               @integration_testing_container
+               container: @integration_testing_container
```

Comment:
I get why we would want to change this, but it would break compatibility, is it possible to avoid this?